contrib/vagrant/provision-master.sh

```diff
@@ -1,72 +1,46 @@
 #!/bin/bash
 
-set -ex
 source $(dirname $0)/provision-config.sh
 
-FIXUP_NET_UDEV=$5
-
-NETWORK_PLUGIN=$(os::util::get-network-plugin ${6:-""})
-
-if [ "${FIXUP_NET_UDEV}" == "true" ]; then
-  NETWORK_CONF_PATH=/etc/sysconfig/network-scripts/
-  rm -f ${NETWORK_CONF_PATH}ifcfg-enp*
-  if [[ -f "${NETWORK_CONF_PATH}ifcfg-eth1" ]]; then
-    sed -i 's/^NM_CONTROLLED=no/#NM_CONTROLLED=no/' ${NETWORK_CONF_PATH}ifcfg-eth1
-    if ! grep -q "NAME=" ${NETWORK_CONF_PATH}ifcfg-eth1; then
-      echo "NAME=openshift" >> ${NETWORK_CONF_PATH}ifcfg-eth1
-    fi
-    nmcli con reload
-    nmcli dev disconnect eth1
-    nmcli con up "openshift"
-  fi
+os::provision::base-provision
+
+os::provision::build-origin "${ORIGIN_ROOT}" "${SKIP_BUILD}"
+os::provision::build-etcd "${ORIGIN_ROOT}" "${SKIP_BUILD}"
+
+echo "Installing openshift"
+os::provision::install-cmds "${ORIGIN_ROOT}"
+os::provision::install-sdn "${ORIGIN_ROOT}"
+
+if [ "${SDN_NODE}" = "true" ]; then
+  # Running an openshift node on the master ensures connectivity between
+  # the openshift service and pods.  This supports kube API calls that
+  # query a service and require that the endpoints of the service be
+  # reachable from the master.
+  #
+  # TODO(marun) This is required for connectivity with openshift-sdn,
+  # but may not make sense for other plugins.
+  NODE_NAMES+=(${SDN_NODE_NAME})
```

Comment:
I don't think "OPENSHIFT_SDN_NODE" is an obvious name for this option anyway, since it doesn't say anything about "master". How about OPENSHIFT_NODE_ON_MASTER or OPENSHIFT_SDN_NODE_ON_MASTER.

(And since our official ansible install also does this, maybe we really ought to make openshift-sdn always set up the SDN on the master as well, without it needing to be made into a node. But that's another bug.)